release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -111,6 +111,20 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-13-upi-scaling-hosts-with-bmo"]
+==== Scale bare-metal hosts in a user-provisioned cluster by using the Bare Metal Operator
+
+With {product-title} {product-version}, you can scale bare-metal hosts in an existing user-provisioned infrastructure cluster by using the Bare Metal Operator (BMO) and other metal3 components. By using the Bare Metal Operator in a user-provisioned cluster, you can simplify and automate the management and scaling of hosts.  
+
+Using the BMO, you can add or remove hosts by configuring a `BareMetalHost` object. Optionally, you can manage existing hosts in the cluster by enrolling them in the `BareMetalHost` object inventory.
```

Comment:
Hi @rohennes , for the optional part, I don't like the wording of `you can manage existing hosts`, this could set wrong expectations for the customers, how about this instead?
~~~
Optionally, you can also keep track of the existing hosts by enrolling them as `externallyProvisioned` in the `BareMetalHost` object inventory.
~~~
Regards.